test/cypress.json

```diff
@@ -1,4 +1,4 @@
 {
-    "baseUrl": "http://nginx:1234",
+    "baseUrl": "http://nginx:3128",
```

Comment:
omfg this is the fix? i didn't understand port forwarding in docker? sdifjadsipofjdsaipofad